lms/lmsdb/bootstrap.py

```diff
@@ -156,7 +156,45 @@ def _upgrade_notifications_if_needed():
     _alter_column_type_if_needed(t, t.message, TextField())
 
 
-def add_solution_state_if_needed():
+def _add_index_if_needed(
+    table: Type[Model],
+    field_instance: Field,
+    is_unique_constraint: bool = False,
+) -> None:
+    column_name = field_instance.name
+    table_name = table.__name__.lower()
+    migrator = db_config.get_migrator_instance()
+    print(f"Add index to '{column_name}' field in '{table_name}'")  # noqa: T001 E501
+    with db_config.database.transaction():
+        try:
+            migrate(
+                migrator.add_index(
+                    table_name, (column_name,), is_unique_constraint,
+                ),
+            )
+        except OperationalError as e:
+            if 'already exists' in e.args[0]:
+                pass
```

Comment:
Add to log as debug/info (print if you want to do it quickly), don't let it pass silently